pkg/vdbgen/vdb.go

```diff
@@ -233,18 +231,18 @@ func (d *DBGenerator) countNodes(ctx context.Context) (int, error) {
 	}
 	defer rows.Close()
 
-	if rows.Err() != nil {
-		return 0, fmt.Errorf("failed running '%s': %w", q, rows.Err())
-	}
-	if !rows.Next() {
-		return 0, errors.New("could not find any nodes in the cluster")
+	if rows.Next() {
```

Comment:
There was code duplication with another method, so I changed this one. I relaxed the checks here as `setSubclusterDetail` is the part handling the node count of each subcluster. Here we only care about Ksafety.